modules/Microsoft.EventHub/namespaces/networkRuleSets/deploy.bicep

```diff
@@ -51,11 +58,11 @@ resource networkRuleSet 'Microsoft.EventHub/namespaces/networkRuleSets@2021-11-0
   name: 'default'
   parent: namespace
   properties: {
-    publicNetworkAccess: !empty(ipRules) || !empty(virtualNetworkRules) ? null : publicNetworkAccess
```

Comment:
If someone sets public access to false, and then specified one of those two properties, wouldn't that break the deployment?

Reply:
So if someone passes in other properties like ipRules or virtualNetworkRules, and also sets publicNetworkAccess to disabled, should that impact the deployment (i.e. let it fail) or should we swap that public network access with null? and let the API handle it?